uint8 LINE=0        #param= [dx,dy,dz]
uint8 SPHERE=1      #param= [rad]
uint8 CYLINDER=2    #param= [rad,len]
uint8 CUBE=3        #param= [lx,ly,lz]

int32 type

geometry_msgs/Pose pose
std_msgs/ColorRGBA color
float64[] param
